geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisSortedSetTest.java

```diff
@@ -147,11 +147,13 @@ public void zadd_stores_delta_that_is_stable() {
     Region<RedisKey, RedisData> region = uncheckedCast(mock(Region.class));
     when(region.put(any(), any())).thenAnswer(this::validateDeltaSerialization);
     RedisSortedSet sortedSet1 = createRedisSortedSet("3.14159", "v1", "2.71828", "v2");
-    List<byte[]> adds = new ArrayList<>();
-    adds.add(stringToBytes("1.61803"));
-    adds.add(stringToBytes("v3"));
+    List<byte[]> members = new ArrayList<>();
+    members.add(stringToBytes("v3"));
+    List<Double> scores = new ArrayList<>();
+    scores.add(1.61803);
```

Comment:
This can be simplified to:
```
    List<byte[]> members = singletonList(stringToBytes("v3"));
    List<Double> scores = singletonList(1.61803);
```